src/index.js

```diff
@@ -56,6 +56,14 @@ class MarkdownShortcuts {
 
     const includeRules = this.options.includeFormats || allRules;
 
+    if (this.options.customFormats) {
+      Object
+        .entries(this.options.customFormats)
+        .forEach(([name, format]) => {
+          this.matches.push(format);
+        });
```

Comment:
May be able to simplify this to `Object.values(this.options.customFormats).forEach(format => this.matches.push(format))` if `this.options.customFormats` is an object as the name isn't used.